frontend/src/lib/rbac-util.ts

```diff
@@ -99,29 +100,32 @@ export function getAuthorizedClusters(resourceAttributes: ResourceAttributes[],
 }
 
 export function checkAdminAccess() {
-    const result = createSubjectAccessReview({
-        name: '*',
-        namespace: '*',
-        resource: '*',
-        verb: '*',
-    }).promise
+    const result = createSubjectAccessReview(
+        Promise.resolve({
+            name: '*',
+            namespace: '*',
+            resource: '*',
+            verb: '*',
+        })
```

Comment:
No need to wrap argument with Promise - `createSubjectAccessReview` accepts either.